### PR TITLE
SC-7040 - landing-page-international-fix

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
fixed theme-blog.js, previously it was not placing html on the page correctly and updated ghost page
## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-7040

## Screenshots of UI changes
image

### CODE REVIEW

**Review by mwolowyk-cap — APPROVED** [review]

File: static/scripts/theme-blog.js [hunk 1]
```diff
@@ -3,9 +3,20 @@ const stripHtml = require('string-strip-html');
 const ghostTheme = $('#SC_THEME').text();
 
 function fetchContent() {
+	let finalHtml = '';
+
 	$(`.${ghostTheme}-blog .spinner`).show();
 	$(`.${ghostTheme}-blog .placeholder`).hide();
 
+	const changePage = () => {
```
Comment (mwolowyk-cap) [inline static/scripts/theme-blog.js:11]: the name is not so descriptive

File: static/scripts/theme-blog.js [hunk 2]
```diff
@@ -3,9 +3,20 @@ const stripHtml = require('string-strip-html');
...
+		finalHtml = finalHtml.replace('<pre><code>', '');
```
Comment (mwolowyk-cap) [inline static/scripts/theme-blog.js:12]: Please don't do tag replacement. Take instead childelement of these tags and set it to finalHtml

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]

File: static/scripts/theme-blog.js (hunk 1 above)
Comment (ABialoskorski) [reply static/scripts/theme-blog.js:11]: done ;)

File: static/scripts/theme-blog.js (hunk 2 above)
Comment (ABialoskorski) [reply static/scripts/theme-blog.js:12]: done, please check ;)

File: static/scripts/theme-blog.js [hunk 3]
```diff
@@ -16,12 +26,8 @@ function fetchContent() {
 		timeout: 8000,
 	})
 		.done((result) => {
-			$(`.${ghostTheme}-blog .title`).text(result.pages[0].title);
-			$(`.${ghostTheme}-blog .content`).html(
-				stripHtml(result.pages[0].html, {
-					onlyStripTags: ['script', 'style'],
-				}),
-			);
+			finalHtml = result.pages[0].html;
```
Comment (mwolowyk-cap) [inline static/scripts/theme-blog.js:29]: why do you need this assignment here and why do you need a global variable here. Probably it is better to do it direct in the function. And finalHtml is not a descriptive name.

**Review by mwolowyk-cap — CHANGES_REQUESTED** [review]

File: static/scripts/theme-blog.js (hunk 3 above)
Comment (ABialoskorski) [reply static/scripts/theme-blog.js]: done ;)

**Review by mwolowyk-cap — APPROVED** [review]

**Pull request merged**